pkg/api/event/hook_response.go

```diff
@@ -200,7 +199,9 @@ type BotProtectionRequirements struct {
 	Mode config.BotProtectionRiskMode `json:"mode,omitempty"`
 }
 
-type RateLimits map[ratelimit.RateLimit]RateLimitRequirements
+// This should be map[ratelimit.RateLimit]RateLimitRequirements,
+// but this will cause import cycle. So use string.
+type RateLimits map[string]RateLimitRequirements
```

Comment:
This isn’t newly added. I’d like to understand why we didn’t have an import cycle issue before.